File: Week_01/G20200343030020/rotate-array.java
class Solution {
    public void rotate(int[] nums, int k) {
        int tmp, pre;
        for (int i = 0; i < k; i++) {
            pre = nums[nums.length - 1];
            for (int j = 0; j < nums.length; j++) {
                tmp = nums[j];
                nums[j] = pre;
                pre = tmp;
            }
        }
    }
}

File: Week_01/G20200343030550/rotate_array.py
# -*- encoding: utf-8 -*-

#暴力破解：超时
def rotateArr(nums,k):
    for i in xrange(k):
        pre = nums[len(nums) - 1]
        for j in xrange(len(nums)):
            temp = nums[j]
            nums[j] = pre
            pre = temp
    return nums





